<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>frainds</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QPushButton" name="testButton">
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QWebEngineView" name="webEngineView">
      <property name="url">
       <url>
        <string>about:blank</string>
       </url>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>807</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuChat">
    <property name="title">
     <string>Chat</string>
    </property>
    <widget class="QMenu" name="menuType">
     <property name="title">
      <string>Type</string>
     </property>
    </widget>
    <addaction name="menuType"/>
   </widget>
   <addaction name="menuChat"/>
  </widget>
  <action name="actionchat1">
   <property name="text">
    <string>chat1</string>
   </property>
  </action>
  <action name="actionchat2">
   <property name="text">
    <string>chat2</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
